emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import os
+from emission.core.wrapper.trip_old import Coordinate
+import emission.net.ext_service.geocoder.nominatim as eco
+
+#temporarily sets NOMINATIM_QUERY_URL to the environment variable for testing.
+eco.NOMINATIM_QUERY_URL = os.environ.get("NOMINATIM_QUERY_URL")
+# print("query URL:", eco.NOMINATIM_QUERY_URL)
```

Comment:
This reads the `NOMINATIM_QUERY_URL` from an environment variable in the test, but monkey patches it into the module.

Reply:
It seems to be a solution to my issue, since I can call and test functions in nominatim.py using the container data without modifying nominatim.json outside of the envirionment. Is there a different/preferred way to do this?

Reply:
once you have changed the module to read from an environment variable (`os.environ.get`)
you can:
- monkey patch in here, OR
- set the environment variable and reload

So for example

```
os.environ.set("...", "http://localhost:...")
importlib.reload(eco)
```

or 

```
os.environ.set("...", "http://localhost:...")
import emission.net.... as eco
```

basically the module should read the environment variable when it is loaded and use it correctly

Reply:
make sure to use fallbacks if the environment variable is not set, just like we use fallbacks if the config file is not present

Reply:
This makes sense for testing. Later on, we will have a secret in secrets manager set the query url environment variable for production, and call in the server with boto3?

Reply:
Conceptually. The cloud team will set the environment variable for the query URL on production using a secure method. I don' t know whether they use boto3 or not, but whatever method they use will be cyber-approved.